core/storage/classifier/gae_models.py

```diff
@@ -218,28 +246,31 @@ def _generate_id(cls, exp_id, exp_version, state_name):
         return utils.convert_to_str(new_id)
 
     @classmethod
-    def get_model(cls, exp_id, exp_version, state_name):
-        """Retrieves the Classifier Exploration Mapping model given Exploration
+    def get_models(cls, exp_id, exp_version, state_names):
+        """Retrieves the Classifier Exploration Mapping models given Exploration
         attributes.
 
         Args:
             exp_id: str. ID of the exploration.
             exp_version: int. The exploration version at the time
                 this training job was created.
-            state_name: unicode. The name of the state to which the classifier
-                belongs.
+            state_names: list(unicode). The state names for which we retrieve
+                the mapping models.
 
         Returns:
-            ClassifierExplorationMappingModel. The model instance for the
+            list(ClassifierExplorationMappingModel). The model instances for the
```

Comment:
It's possible that some entries could be None, right? This should be part of the documentation and there should be unit tests for this case.